test/unit/examples/oauthRequests/getTokenFromTrustKeyrockGroup.json

```diff
@@ -0,0 +1 @@
+client_id=context-broker&client_secret=c8d58d16-0a42-400e-9765-f32e154a5a9e&grant_type=refresh_token&refresh_token=8970A9078A803H3BL98PINEQRW8342HBAMS
```

Comment:
I understand the tokens used in this an other examples in this PR ar fake, isn't it?

Reply:
NTC